andela/ah-frontend-sparta
# #164798167 Enable users to view and edit their profiles

#### What does this PR do?
Enable users to view and edit their profiles
#### Description of Task to be completed?
- Enable user to view profile
- Enable user to edit profile
- Enable user to upload their image

#### How should this be manually tested?
- Fetch this branch git fetch origin `ft-user-should-view-and-edit-their-profiles-164798167`
- Checkout to the branch git checkout `ft-user-should-view-and-edit-their-profiles-164798167`
- Run npm start to view the landing page
- Login to the app on `/Login`
- View profile on `/profile`
- Edit profile on `/editprofile`

#### What are the relevant pivotal tracker stories?
[#164798167]( https://www.pivotaltracker.com/story/show/164798167)

#### Screenshots (if appropriate)
<img width="1368" alt="Screenshot 2019-05-16 at 23 27 40" src="https://user-images.githubusercontent.com/35653907/57884866-57e78780-7832-11e9-9643-d568621db95a.png">
<img width="1366" alt="Screenshot 2019-05-16 at 23 27 54" src="https://user-images.githubusercontent.com/35653907/57884885-62098600-7832-11e9-8dc7-90764bc4531e.png">
